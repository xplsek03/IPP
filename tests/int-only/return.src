<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode19">
 <instruction order="1" opcode="WRITE">
  <arg1 type="string">a</arg1>
 </instruction>
 <instruction order="2" opcode="CALL">
  <arg1 type="label">f1</arg1>
 </instruction>
 <instruction order="3" opcode="WRITE">
  <arg1 type="string">b</arg1>
 </instruction>
 <instruction order="4" opcode="CALL">
  <arg1 type="label">f2</arg1>
 </instruction>
 <instruction order="5" opcode="WRITE">
  <arg1 type="string">c</arg1>
 </instruction>
 <instruction order="6" opcode="CALL">
  <arg1 type="label">konec</arg1>
 </instruction>
 <instruction order="7" opcode="LABEL">
  <arg1 type="label">f1</arg1>
 </instruction>
 <instruction order="8" opcode="WRITE">
  <arg1 type="string">f1</arg1>
 </instruction>
 <instruction order="9" opcode="RETURN"/>
 <instruction order="10" opcode="LABEL">
  <arg1 type="label">f2</arg1>
 </instruction>
 <instruction order="11" opcode="WRITE">
  <arg1 type="string">f2</arg1>
 </instruction>
 <instruction order="12" opcode="CALL">
  <arg1 type="label">f2_2</arg1>
 </instruction>
 <instruction order="13" opcode="RETURN"/>
 <instruction order="14" opcode="LABEL">
  <arg1 type="label">f2_2</arg1>
 </instruction>
 <instruction order="15" opcode="WRITE">
  <arg1 type="string">f2_2</arg1>
 </instruction>
 <instruction order="16" opcode="RETURN"/>
 <instruction order="17" opcode="LABEL">
  <arg1 type="label">konec</arg1>
 </instruction>
 <instruction order="18" opcode="WRITE">
  <arg1 type="string">d</arg1>
 </instruction>
</program>
