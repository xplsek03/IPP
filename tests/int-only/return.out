af1bf2f2_2cd